src/docs/patterns/overlay.mdx

```diff
@@ -1,73 +1,132 @@
+They can also present simple form data and inputs (such as radio buttons, checkbox, dropdown menus, etc.) for editing a selected item. This type of sidebar should be temporary such that the user can easily close it with a close button or clicking elsewhere on the screen.
 
-In general, modals and sidebars can be used together to set up and display complex data. When building your application, you may be faced with the question of "Should I use a modal or a sidebar for this?" In general, modals should be used when creating new data, and sidebars should be used for viewing and editing existing data. In some circumstances, you may need a full-page overlay (sidebar or bottomsheet) to accomplish more complex tasks.
+Sidebars should always appear on the opposite side of the screen as the main [navigation](/patterns/navigation).
 
-## Bottom Sheet
+> In general, modals should be used when creating new data, and sidebars should be used for viewing and editing existing data.
 
-A bottom sheet is similar to a modal dialog but slides in from the bottom of the screen. A dark overlay is usually used to block user interaction with the main app until the bottom sheet is closed.
+## Bottom Sheet
 
 <MaterialDesignDescription title={'Sheets: Bottom'} url={'https://material.io/components/sheets-bottom/'} />
 
-This is a very common pattern in mobile applications but can also be used in responsive web apps. Typically, the bottom sheet contains global actions for the entire page and is activated by a button in the app bar.
+Bottom sheets are content surfaces that slide in from the bottom of the screen, primarily used in mobile applications. There are three types: standard, modal, and expandable. The content shown in a bottom sheet can be as simple as a list of items or as complex as a full screen layout.
 
-<ImageGrid images={[BasicBottomsheet, ComplexBottomsheet]} caption={'A Bottom Sheet can be simple or complex.'} />
+### Standard
 
-### Basic
+A standard bottom sheet is always visible and non-blocking, used most commonly for multitasking, such as displaying media player controls while browsing other content.
 
-<DemoCard
-    repository={'bottomsheet'}
-    angular={
-        'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Foverlays%2Fbasic-bottom-sheet%2Fbasic-bottom-sheet.component.ts'
-    }
-    react={
-        'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/overlays/basic-bottom-sheet/index.tsx&initialpath=basic-bottom-sheet'
-    }
-    ionic
-    reactNative
-/>
+### Modal
 
-The basic bottom sheet simply contains a list of actions. The final "Cancel" action is used to dismiss the bottom sheet.
+<React.Fragment>
+    <ImageGrid
+        regularWidth
+        images={[SimpleBottomsheet]}
+        caption={[
+            '1. On mobile, the bottom sheet takes up the full width of the screen.',
+            '2. On desktop, the bottom sheet be no more than 600px wide.',
```

Comment:
I 100% agree that this is what a bottom sheet should look like on a desktop. But are we _encouraging_ people to use a bottom sheet on the desktop? Or should we push them to use menus instead?

Reply:
Are menus another form of overlays?

Reply:
I agree this is a bad example. I will update the image to use a menu for desktop.